docs/quick-start.md

```diff
@@ -2,63 +2,64 @@
 
 This document will go over the quickest way to get this platform up and running. Here we will choose the simplest approach for automation and hosting, which is to use GitHub. Note, however, that there are alternatives to this approach, as detailed under the Automation and Hosting sections.
 
-> #### Need a quicker quick start?
-> This document recommends a double-repository approach,
-> which separates your platform into a "site repository" and a "data repository".
-> This is a good practice in general, and provides several logistical benefits.
-> However, a simpler single-repository approach may be preferred if you just
-> want to try out Open SDG locally. In this case, see
-> the [open-sdg-simple-starter project](https://github.com/open-sdg/open-sdg-simple-starter).
-
 ## Signing up and creating repositories
 
 1. If you don't already have a Github.com account, [go to Github.com](https://github.com/) to sign up and then log in.
 1. Go to the [site starter](https://github.com/open-sdg/open-sdg-site-starter) and click the green "Use this template" button.
-1. Next you will be prompted to choose a name for your new repository. This will affect the URL at which you access the site later, so choose carefully. A suggestion might be: `sdg-site-australia` (adjusted for your country). Note that you can change this later if needed.
-1. Enter a description if you would like. Leave "Public" selected, check the "Include all branches" box, and click "Create repository from template".
-    * Bookmark the created repository -- this is your "__site repository__".
+1. You can enter any name for the repository. Here we recommend using "site".
```

Comment:
@brockfanning I thought that they would have to name the repos open-sdg-site-starter and open-sdg-data-starter in order for the site to build without any changes being needed?

Reply:
Ah I see - you have a section below about connecting the site and data

Reply:
Sounds great!